### PR TITLE
Pos design updates #1

### PR DESCRIPTION
**Pull request by stiaan**
Changes proposed in this pull request:

* 
* 
* 

### Reviewer, please refer to this "definition of done" checklist:

* [ ] Tested on **desktop** (see [supported browsers](https://www.lightningdesignsystem.com/faq/#what-browsers-are-supported))
* [ ] Tested on **mobile** (for responsive or mobile-specific features)
* [ ] Confirm **Accessibility**
* [ ] Documentation is up to date
* [ ] Release notes mention the changes

⚠️ Once this pull request is merged, please merge the code into other development branches:
[Merge branch 'spring-17' into summer-17](http://bit.ly/2fjT4LY)
[Merge branch 'summer-17' into winter-18](http://bit.ly/2mbKAbV)

### CODE REVIEW

**Comment by stiaan** [comment]
@NicolaasDieBaas Please review

File: ui/dependencies/_typography.scss [hunk 1]
```diff
@@ -41,3 +41,8 @@
   text-transform: uppercase;
   letter-spacing: 0.0625rem;
 }
+
+// Added for Picsa OS for better display of Proxima Nova
```
Comment (NicolaasDieBaas) [inline ui/dependencies/_typography.scss]: @stiaan does this act as a type of placeholder comment for future changes..?  If not, you can remove

**Comment by NicolaasDieBaas** [comment]
@stiaan I think you can remove all 'POS' comments before landing this issue as they have served their purpose of giving me an understanding of why changes were made.  Do you agree?

**Comment by NicolaasDieBaas** [comment]
@stiaan after the necessary changes has been made, you can land this issue. 👍

**Pull request merged**